.circleci/config.yml

```diff
@@ -378,6 +378,38 @@ jobs:
           env: "production"
           notify_slack: "true"
 
+  ecr-cleanup:
+    executor: cloud-platform-executor
+    steps:
+      - run:
+          name: Check for too many images
+          command: |
+            COUNT=$(aws ecr list-images --repository-name 'book-a-secure-move/hmpps-book-secure-move-api' | jq '.imageIds | unique_by(.imageDigest) | length')
+            if [ "$COUNT" -lt 200 ]; then circleci step halt; exit 0;  fi
+            echo "More than 200 images found. Proceeding with cleanup"
+      - run:
+          name: Build manifest
+          command: |
+            TO_DELETE=$(aws ecr describe-images --region=eu-west-2 \
+              --repository-name book-a-secure-move/hmpps-book-secure-move-api \
+              --query 'imageDetails[]' --output json |\
+              jq 'sort_by(.imagePushedAt)|map(select(any(.imageTags[]? ; contains("latest"))|not))|map({imageDigest})|.[:100]')
+            echo "Tags to be deleted:"
+            aws ecr describe-images --region=eu-west-2 \
+              --repository-name book-a-secure-move/hmpps-book-secure-move-api \
+              --image-ids "${TO_DELETE}" --output json  | jq '.[][].imageTags[]'
+            echo "Deleting:"
+            aws ecr batch-delete-image --region eu-west-2 \
+            --repository-name book-a-secure-move/hmpps-book-secure-move-api \
+            --image-ids ${TO_DELETE}
+      - run:
+          name: Confirm live latest images
+          command: |
```

Comment:
Display the images whose tags contain "latest" that were left after the run. 
Just for peace of mind